model/_global/ping/operations.smithy

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: Apache-2.0
+// 
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+namespace OpenSearch
+
+@readonly
+@suppress(["HttpUriConflict"])
+@http(method: "GET", uri: "/")
+@documentation("It checks whether the cluster is up and available to process requests.")
```

Comment:
Can we change this to the description used here: https://github.com/opensearch-project/OpenSearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/ping.json